Examples/SubscriberExample/Sources/Screens/Map/MapViewController.swift

```diff
@@ -224,4 +224,8 @@ extension MapViewController: SubscriberDelegate {
         assetStatusLabel.textColor = statusDescAndColor.color
         assetStatusLabel.text = statusDescAndColor.desc
     }
+    
+    func subscriber(sender: Subscriber, didUpdateEnhancedLocation resolution: Resolution) {
+        // TODO
+    }
```

Comment:
Why is it added here? :thinking: Also the second param name probably shouldn't be `didUpdateEnhancedLocation` :sweat_smile: